website/profile/views.py

```diff
@@ -408,6 +409,44 @@ def oauth_application_detail(auth, **kwargs):
     return {"app_list_url": '',
             "app_detail_url": app_detail_url}
 
+@must_be_logged_in
+def personal_access_token_list(auth, **kwargs):
+    """Return token creation page with list of known tokens. API is responsible for tying list to current user."""
+    token_list_url = api_v2_url("tokens/")
+    return {
+        "token_list_url": token_list_url
+    }
+
+@must_be_logged_in
+def personal_access_token_register(auth, **kwargs):
+    """Register an API personal token: blank form view"""
```

Comment:
These are not API tokens; these are OAuth2 tokens. Please update docstring.  

Pedantic, but it helps encourage good code organization: we should recognize that the same authentication system might in the future be used across many COS services, not just the OSF APIv2.